src/nvapi_gpu.cpp

```diff
@@ -595,6 +595,8 @@ extern "C" {
                 implementationId = NV_GPU_ARCH_IMPLEMENTATION_GP102;
                 break;
             case NV_GPU_ARCHITECTURE_GM200:
+            case NV_GPU_ARCHITECTURE_GM000:
+                // There is no NV_GPU_ARCH_IMPLEMENTATION_GM107
```

Comment:
I'm now curious how native NVAPI reports it.

Reply:
If I had to guess it would be `0x00000007` considering the pattern of the values in `NV_GPU_ARCH_IMPLEMENTATION_ID` ;)